src/routes/user/userRoute.js

```diff
@@ -0,0 +1,26 @@
+import { createUser, userLogin } from '../../controllers/user.controllers';
```

Comment:
Maybe the name of the file could be `user.routes.js` so that it is more consistent with the names of the other files containing routes?